### PR TITLE
Added dedicated methods and properties for media devices

### LINKED ISSUE #24: Media devices support
Would it be possible to add support for media devices (TVs, soundbars, etc,)?
It seems that there already is some basic support for them (appropriate capabilities and attribute names), but there are no dedicated methods for executing commands nor attributes to retrieve current value.

https://smartthings.developer.samsung.com/docs/api-ref/capabilities.html#Media-Playback

### LINKED ISSUE #27: Change mediaInputSource on SmartTV
How can I change the mediaInputSource on my SmartTV?
Im new in programming with smartthings, so i don't get your doku.
Regards,
Felix

### PR DESCRIPTION
**Pull request by PiotrMachowski**
## Description:
Added dedicated methods and properties for media devices. 

**Related issue (if applicable):** 
fixes #24
fixes #27 

## Checklist:
  - [x] The code change is tested and works locally.
  - [x] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [x] There is no commented out code in this PR.
  - [x] Tests have been added/updated and code coverage percentage does not drop. No exclusions in `.coveragerc` allowed
  - [x] `README.MD` updated (if necessary)

### CODE REVIEW

**Comment by andrewsayre** [comment]
Make sure you're formatting the code with `black` :)

**Comment by PiotrMachowski** [comment]
That's much easier 😃

**Comment by PiotrMachowski** [comment]
Is there any specific reason to keep so many classes (`Command`, `Device`, `DeviceStatusBase`, `DeviceStatus`, `DeviceEntity`) in one big file (`device.py`?

**Comment by PiotrMachowski** [comment]
@andrewsayre any progress here?

**Comment by PiotrMachowski** [comment]
@andrewsayre

**Comment by andrewsayre** [comment]
Sorry for the delay, I will review this week and move this forward.

**Comment by PiotrMachowski** [comment]
No pressure :)

File: pysmartthings/device.py [hunk 1]
```diff
@@ -185,6 +201,13 @@ def is_on(self, attribute: str) -> bool:
             return bool(self._attributes[attribute].value)
         return self._attributes[attribute].value == ATTRIBUTE_ON_VALUES[attribute]
 
+    @staticmethod
```
Comment (andrewsayre) [inline pysmartthings/device.py:204]: There's no need for this to be a static method given it doesn't reference anything within the class.  Let's make this a module level function.

**Comment by PiotrMachowski** [comment]
Thanks!

**Comment by PiotrMachowski** [comment]
You can check out my updated version of SmartThings integration for media players [here](https://github.com/PiotrMachowski/Home-Assistant-test-components-SmartThings/blob/master/custom_components/smartthings/media_player.py).

**Pull request merged**